core/src/main/java/hudson/os/SU.java

```diff
@@ -88,10 +88,11 @@ protected Process sudoWithPass(ArgumentListBuilder args) throws IOException {
                     Process p = pb.start();
                     // TODO: use -p to detect prompt
                     // TODO: detect if the password didn't work
-                    PrintStream ps = new PrintStream(p.getOutputStream());
-                    ps.println(rootPassword);
-                    ps.println(rootPassword);
-                    ps.println(rootPassword);
+                    try (PrintStream ps = new PrintStream(p.getOutputStream())) {
```

Comment:
That one still concerns me, because we have no test coverage for it AFAICT

Reply:
Ok, I reverted it for know and we later deal with that finding.